control/exception.py

```diff
@@ -46,7 +46,9 @@ class ControlSlycot(Exception):
 
 class ControlDimension(Exception):
     """Raised when dimensions of system objects are not correct"""
-    pass
```

Comment:
As discussed by @repagh in PR #268, this change has side effects.  Suggest leaving this alone since the default behavior should be enough.